include/rtl_433_devices.h

```diff
@@ -12,7 +12,6 @@
 		DECL(elv_em1000) \
 		DECL(elv_ws2000) \
 		DECL(lacrossetx) \
-		DECL(acurite5n1) \
```

Comment:
You need to register the template decoder here again. Otherwise you will mess up ordering and people will have to change their scripts.